controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescription.java

```diff
@@ -242,7 +289,7 @@ private String parseAttributes(final XMLExtendedStreamReader reader, ModelNode o
         return res;
     }
 
-    private void parseChildren(final XMLExtendedStreamReader reader, PathAddress parentAddress, List<ModelNode> list, ModelNode op) throws XMLStreamException {
+    public void parseChildren(final XMLExtendedStreamReader reader, PathAddress parentAddress, List<ModelNode> list, ModelNode op) throws XMLStreamException {
```

Comment:
Why public?

Reply:
good catch, it was used in some experiments, but forgot to revert the change. Fixed